app/assets/javascripts/ui_models/app_state/app_state.ts

```diff
@@ -80,6 +80,7 @@ export class AppState {
     application: WebApplication,
     private bridge: Bridge
   ) {
+    console.log('enableUnfinishedFeatures', this.enableUnfinishedFeatures);
```

Comment:
```suggestion
```

Reply:
Fixed.

Reply:
Should we maybe only print this log on the dev app?

Reply:
I removed that, we don't need it. :D I'm gonna resolve this conversation.